quickstart/ia32-generic-qemu.md

```diff
@@ -124,6 +124,141 @@ The number of detected cores is presented during kernel initialization.
 
 ![Image](_images/qemu-ia32-generic-smp.png)
 
+## Network setup on `ia32-generic-qemu`
+
+- Note: This guide was tested on `Ubuntu 20.04 LTS` host OS.
+
+There are few steps to follow:
+
+ 1. Create and set up `vibr0` bridge on a host using `qemu-bridge-helper`:
+
+    - Install the required package and ensure that `libvirtd` is running:
+
+      ```bash
+      sudo apt-get update
+      sudo apt-get install qemu-system-common
```

Comment:
```
sudo apt update
sudo apt install qemu-system-common
```

missing information about libvirtd instalation

Reply:
I think now it is complete